src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -102,6 +102,48 @@ class PdlClient(
         }
     }
 
+    suspend fun hentIdenter(
```

Comment:
Er det sikkert at vi trenger å gjøre et eget kall for å hente identer for person?. Ser at [Person](https://github.com/navikt/pdl/blob/master/apps/api/src/main/resources/schemas/pdl.graphqls#L133)-objektet man får med `hentPerson` også har felt for `folkeregisteridentifikator` (inkludert historikk?) så kanskje det går an å bare utvide `hentPerson`-queriet :thinking:

Reply:
Kan kanskje funke, men usikker på hvordan det api'et fungerer med gammel/utgått personident. hentIdenter skal i alle fall fungere med både gamle/utgåtte og nye/aktive identer.

Reply:
Hm, fant denne på [Slack](https://nav-it.slack.com/archives/C84H68ESC/p1706703362388399), der virker det som det fungerer. Mulig det kan testes med Dolly også :thinking:

Reply:
Det finnes visst en https://pdl-playground.dev.intern.nav.no/editor :nerd_face: Prøvd `hentPerson` på `04477621991` nå (som Eirik bytta ident på forrige uke - ny ident er `02487624384`) og det funka.

Reply:
Fikset nå (tror jeg).